assets/js/interactivity/router.js

```diff
@@ -100,8 +106,9 @@ const fetchAssets = async ( document ) => {
 };
 
 // Fetch a new page and convert it to a static virtual DOM.
-const fetchPage = async ( url ) => {
-	const html = await window.fetch( url ).then( ( r ) => r.text() );
+const fetchPage = async ( url, options = {} ) => {
+	const html =
+		options.html || ( await window.fetch( url ).then( ( r ) => r.text() ) );
```

Comment:
We can revert all the router changes in this PR because they are not needed anymore. We'll add them directly to the Interactivity API.

Reply:
Can I just restore the files that are currently in the trunk?

Reply:
Yes. We'll add those changes upstream and then we'll update them here in a separate PR to keep things in context 🙂